gpkit/__init__.py

```diff
@@ -56,11 +56,24 @@ def enableUnits():
     except ImportError:
         print("Optional Python units library (Pint) not installed;"
               "unit support disabled.")
-        disableUnits()
+        disable_units()
 
-enableUnits()
+enable_units()
+
+_SIGNOMIALS_ENABLED = False
+
+
+def enable_signomials():
+    """Enables signomial support in a particular instance of GPkit."""
+    global _SIGNOMIALS_ENABLED
+    _SIGNOMIALS_ENABLED = True
+
+
+def disable_signomials():
+    """Disables signomial support in a particular instance of GPkit."""
+    global _SIGNOMIALS_ENABLED
+    _SIGNOMIALS_ENABLED = False
```

Comment:
From pep8:

Constants
---------
Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include `MAX_OVERFLOW` and `TOTAL` .

Global Variable Names
Modules that are designed for use via from M import * should use the __all__ mechanism to prevent exporting globals, or use the older convention of prefixing such globals with an underscore (which you might want to do to indicate these globals are "module non-public").